dev-perl/DateTime-Format-Human-Duration/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>c.affolter@stepping-stone.ch</email>
+		<name>Christian Affolter</name>
+		<description>Maintainer</description>
```

Comment:
The `<description />` tag is in the [metadata.xml](https://github.com/gentoo/gentoo/blob/master/www-apps/webdavcgi/metadata.xml) of the old webdavcgi ebuild, that's why I added it here as well. Do you want me to remove it?